lambdas/session/src/main/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionAnswerHandler.java

```diff
@@ -8,10 +8,7 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import software.amazon.lambda.powertools.tracing.CaptureMode;
 import software.amazon.lambda.powertools.tracing.Tracing;
-import uk.gov.di.ipv.cri.kbv.api.domain.QuestionAnswer;
-import uk.gov.di.ipv.cri.kbv.api.domain.QuestionAnswerRequest;
-import uk.gov.di.ipv.cri.kbv.api.domain.QuestionState;
-import uk.gov.di.ipv.cri.kbv.api.domain.QuestionsResponse;
+import uk.gov.di.ipv.cri.kbv.api.domain.*;
```

Comment:
* you used star import here see https://gds-way.cloudapps.digital/manuals/programming-languages/java.html#imports